rest_framework/templatetags/rest_framework.py

```diff
@@ -2,23 +2,99 @@
 
 import re
 
+from collections import OrderedDict
 from django import template
 from django.template import loader
 from django.utils import six
 from django.utils.encoding import force_text, iri_to_uri
 from django.utils.html import escape, format_html, smart_urlquote
 from django.utils.safestring import SafeData, mark_safe
+from markdown.extensions.fenced_code import FencedBlockPreprocessor
```

Comment:
This import makes `markdown` non-optional. c.f. https://github.com/carltongibson/django-filter/pull/658